DOCUMENTATION.md

```diff
@@ -849,6 +893,239 @@ which serves to add tasks for Chainlink Keepers to perform on client contracts.
 - **Arguments:**
   - `keeperRegistryAddress`: Address of Keeper Registry
 
+## Functions Services
```

Comment:
Can we document return types as well?

Reply:
Yes, definitely! Thank you for mentioning it.